script.xbmcbackup/resources/.gitignore

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
still here

Reply:
:( I'm not sure why there is even an ignore file here, serves no purpose.